spec/acceptance/ini_setting_spec.rb

```diff
@@ -42,52 +41,57 @@
   end
 
   context 'ensure parameter => present for global and section' do
+    basedir = setup_test_directory
```

Comment:
Use a single `let()` in the previous context to define `basedir`.

Reply:
As it is now, we can't declare basedir variable with let, because it can't be used inside describe/context blocks or to pass it to shared_example block.
I've added at the beginning of the tests: basedir = setup_test_directory and
removed the duplicates